src/Microsoft.OData.Edm/Vocabularies/CommunityVocabularies.xml

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<edmx:Edmx xmlns:edmx="http://docs.oasis-open.org/odata/ns/edmx" Version="4.0">
+  <edmx:Reference Uri="http://docs.oasis-open.org/odata/odata/v4.0/os/vocabularies/Org.OData.Core.V1.xml">
+    <edmx:Include Alias="Core" Namespace="Org.OData.Core.V1" />
+  </edmx:Reference>
+  <edmx:DataServices>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OData.Community.Keys.V1" Alias="Keys">
+      <Term AppliesTo="EntityType" Type="Collection(Keys.AlternateKey)" Name="AlternateKeys">
+        <Annotation Term="Core.Description" String="Communicates available alternate keys"/>
+      </Term>
+      <ComplexType Name="AlternateKey">
+        <Property Type="Collection(Keys.PropertyRef)" Name="Key">
+          <Annotation Term="Core.Description" String="The set of properties that make up this key"/>
+        </Property>
+      </ComplexType>
+      <ComplexType Name="PropertyRef">
+        <Property Type="Edm.PropertyPath" Name="Name">
+          <Annotation Term="Core.Description" String="A path expression resolving to a primitive property of the entity type itself or to a primitive property of a complex property (recursively) of the entity type. The names of the properties in the path are joined together by forward slashes."/>
+        </Property>
+        <Property Type="Edm.String" Name="Alias">
+          <Annotation Term="Core.Description" String="A SimpleIdentifier that MUST be unique within the set of aliases, structural and navigation properties of the containing entity type that MUST be used in the key predicate of URLs"/>
+        </Property>
+      </ComplexType>
+    </Schema>
+    <Schema xmlns="http://docs.oasis-open.org/odata/ns/edm" Namespace="OData.Community.UrlEscape.V1" Alias="UrlEscape">

```

Comment:
The name space and the alias should be better, not only for the "UrlEscape". I think we will add more into this schema for more community terms. Check with @mikepizzo for better naming.

Reply:
As we discussed, I have moved the UrlEscapeFunction term into a new CommunityVocabularies.xml, keeping existing AlternativeKeysVocabularies.xml unchanged (will be deprecated later).

Reply:
Resolved: have updated namespace and alias to community.

---
In reply to: [229003178](https://github.com/OData/odata.net/pull/1307#discussion_r229003178) [](ancestors = 229003178)